installer/charts/rhtap-acs-scanner/scripts/test-acs-image-scan.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROX_CENTRAL_ENDPOINT="$(kubectl get secrets -n rhtap rhtap-acs-integration -o jsonpath='{.data.endpoint}' | base64 -d)"
+ROX_API_TOKEN="$(kubectl get secrets -n rhtap rhtap-acs-integration -o jsonpath='{.data.token}' | base64 -d)"
```

Comment:
You need to mount this secret on the testing POD, by doing so the `ROX_CENTRAL_ENDPOINT` and `ROX_API_TOKEN` will be available directly in the environment.